gateway/src/cluster/builder.rs

```diff
@@ -39,21 +37,25 @@ use twilight_model::gateway::{
 /// ```
 ///
 /// [`large_threshold`]: Self::large_threshold
-#[derive(Debug)]
-pub struct ClusterBuilder(ClusterConfig, ShardBuilder);
+pub struct ClusterBuilder {
```

Comment:
can we add a comment here reminding if developers add a field, they should add it to the debug implementation